trs_filer/config.yaml

```diff
@@ -13,13 +13,18 @@ db:
         trsStore:
             collections:
                 objects:
-                    indexes: null
+                    indexes:
+                        - keys:
+                              id: 1
+                          options:
+                            'unique': True
 
 api:
     specs:
         - path:
             - api/20200404.0aa5443.openapi.yaml
             - api/20200131.11d2c12.service-info.yaml
+            - api/additional_post_endpoints.openapi.yaml
```

Comment:
maybe just call it `additions.openapi.yaml`, otherwise you would need to change it once you put in some endpoint that is not a `POST`